examples/deploy_specific_keys.yml

```diff
@@ -0,0 +1,16 @@
+---
+# This playbook will deploy a specific set of keys to every NBDE server.
+# Sharing keys is not recommended, but this is supported. To do this, place
+# the keys in the top level directory of nbde_server_keys_dir, then run this
+# playbook.
+- hosts: all
+  become: true
+
+  vars:
+    nbde_server_fetch_keys: yes
+    nbde_server_deploy_keys: yes
+    nbde_server_keys_dir: /root/nbde_server/keys
```

Comment:
How is example different from the `deploy_and_reuse_keys.yml` example?

Reply:
Hm. It looks redundant, indeed. I removed this one.